files/fail_over.py

```diff
@@ -0,0 +1,167 @@
+                                                   var_file=None))
+        ansible_play = settings.get(_SECTION, _ANSIBLE_PLAY,
+                                    vars=DefaultOption(settings,
+                                                       _SECTION,
+                                                       ansible_play=None))
+        while target_host not in setups:
+            target_host = raw_input(
+                INPUT + PREFIX + "target host was not defined. "
+                "Please provide the target host "
+                "(primary or secondary): " + END)
+        while source_map not in setups:
+            source_map = raw_input(
+                INPUT + PREFIX + "source mapping was not defined. "
+                "Please provide the source mapping "
+                "(primary or secondary): " + END)
+        while not os.path.isfile(var_file):
+            var_file = raw_input("%s%svar file mapping '%s' does not exist. "
+                                 "Please provide a valid mapping var file: %s"
+                                 % (INPUT,
+                                    PREFIX,
+                                    var_file,
+                                    END))
+        while not os.path.isfile(vault):
+            vault = raw_input("%s%spassword file '%s' does not exist."
+                              "Please provide a valid password file:%s "
+                              % (INPUT.
+                                 PREFIX,
+                                 vault,
+                                 END))
+        while (not ansible_play) or (not os.path.isfile(ansible_play)):
+            ansible_play = raw_input("%s%sansible play '%s' "
+                                     "is not initialized.\n "
+                                     "Please provide the ansible play file "
+                                     "to generate the mapping var file "
+                                     "with ('%s'):%s" or PLAY_DEF
+                                     % (INPUT,
+                                        PREFIX,
+                                        str(ansible_play),
+                                        PLAY_DEF,
+                                        END))
+        return (target_host, source_map, var_file, vault, ansible_play)
+
+
+class DefaultOption(dict):
+
+    def __init__(self, config, section, **kv):
+        self._config = config
+        self._section = section
+        dict.__init__(self, **kv)
+
+    def items(self):
+        _items = []
+        for option in self:
+            if not self._config.has_option(self._section, option):
+                _items.append((option, self[option]))
+            else:
+                value_in_config = self._config.get(self._section, option)
+                _items.append((option, value_in_config))
+        return _items
+
+
+if __name__ == "__main__":
+    FailOver().run('dr.conf', '/var/log/ovirt-dr/ovirt-dr.log')
```

Comment:
/var/log/ovirt-dr/ is this directory created by RPM? I think it won't work from galaxy

Reply:
no it is a directory which I'm creating.
How I should do it then?

Reply:
in the build.sh script would be simplest

Reply:
I've found ovirt-engine.spec.in and add the log as mention inthere.
It can be reviewed in the new commit